### ISSUE TITLE
Test accessibility of dialogs using screen readers

### ISSUE THREAD

**Issue by terrill** [ableplayer/ableplayer#80]
Using JAWS 15 in IE11, when a dialog opens JAWS doesn't announce it, nor can it finds its contents.  Also the modal dialog still doesn't seem to modal (can interact with background using JAWS), and escape doesn't close the dialog. 

Need further testing using other screen reader/browser combinations.
